bin/paragon-scripts.js

```diff
@@ -173,6 +174,8 @@ const COMMANDS = {
   const [command, ...commandArgs] = process.argv.slice(2);
   const executor = COMMANDS[command];
 
+  sendTrackInfo('openedx.paragon.cli-command.used', { command });
```

Comment:
Hmm, I think it might be a good idea to add additional info about the command, for example if it succeeded or not. What do you think about moving this code into try / catch block below and sending status of execution?
Something like:
```js
try {
    await executor.executor(commandArgs);
    sendTrackInfo('openedx.paragon.cli-command.used', { command, status: 'success' });
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error(chalk.red.bold('An error occurred:', error.message));
    sendTrackInfo('openedx.paragon.cli-command.used', { command, status: 'error', errorMsg: error.message });
    process.exit(1);
  }
```

Reply:
added, thanks